homeassistant/components/mobile_app/webhook.py

```diff
@@ -137,9 +144,26 @@ def setup_registration(hass: HomeAssistantType, store: Store,
                                  headers=headers)
 
     if webhook_type == WEBHOOK_TYPE_UPDATE_LOCATION:
+        see_payload = {
+            ATTR_DEV_ID: registration[ATTR_DEVICE_ID],
+            ATTR_HOST_NAME: registration[ATTR_DEVICE_NAME],
+            ATTR_LOCATION_NAME: data.get(ATTR_LOCATION_NAME),
+            ATTR_GPS: data.get(ATTR_GPS),
+            ATTR_GPS_ACCURACY: data.get(ATTR_GPS_ACCURACY),
+            ATTR_BATTERY: data.get(ATTR_BATTERY),
+            ATTR_SOURCE_TYPE: data.get(ATTR_SOURCE_TYPE),
+            ATTR_CONSIDER_HOME: data.get(ATTR_CONSIDER_HOME),
```

Comment:
FYI: We're talking about dropping support for consider home in device tracker.

Reply:
We should completely drop it here, we should not support source type/consider home. It's always GPS?

Reply:
Existing discussion on `source_type` is at https://github.com/home-assistant/home-assistant/issues/21761.

Short version is that apps on iOS and Android can have location updates triggered for more than just GPS (both platforms support GPS, Bluetooth and Bluetooth Low Energy, e.g. iBeacon and Eddystone updates).

Reply:
I'm fine dropping `consider_home`.

Reply:
After hearing feedback from others, I'll drop `source_type` too and figure out a workaround for the iOS app separately.

Reply:
`source_type` is planned to be a mandatory device tracker entity property after the device tracker refactor.